### ISSUE TITLE
[wg/social] Social Web Working Group new charter

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#435]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/charter-drafts/2025/social-wg.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/charter-drafts/2025/social-wg.html)

[diff with charter proposed by CG](https://services.w3.org/htmldiff?doc1=https://swicg.github.io/potential-charters/social-web-wg-charter.html&doc2=https://w3c.github.io/charter-drafts/2025/social-wg.html)

Expected end of charter refinement phase: June

If applicable:

[diff from previous 1018 group charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2013%2Fsocialweb%2Fsocial-wg-charter&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fsocial-wg.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/social)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [x] New WG 

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

Social Web CG.

Known or potential areas of concern: 

lack of consensus within the Community on the scope of the charter.

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

Use [w3c/charter-drafts](https://github.com/w3c/charter-drafts/issues)

Anything else we should think about as we review? 

Nope

**Comment by plehegar** [comment]
Notes:
* Social Web WG or ActivityPub WG ?
* Maintenance-only or allow new features?
* Make it an umbrella WG around a CG?

**Comment by gobengo** [comment]
@plehegar noting that at this time there have been two contributors to that document

**Comment by plehegar** [comment]
[[
Note: If there is no [Working Group](https://www.w3.org/2023/Process-20230612/#GroupsWG) chartered to maintain a [Recommendation](https://www.w3.org/2023/Process-20230612/#RecsW3C) the [Team](https://www.w3.org/2023/Process-20230612/#team) cannot make substantive changes and republish the [Recommendation](https://www.w3.org/2023/Process-20230612/#RecsW3C). It can, however, informatively highlight problems and desirable changes using [errata](https://www.w3.org/2023/Process-20230612/#erratum) and [candidate corrections](https://www.w3.org/2023/Process-20230612/#candidate-correction) and republish as described in [the previous section](https://www.w3.org/2023/Process-20230612/#revised-rec-editorial).
]]
From [6.3.11.3. Revising a Recommendation: Substantive Changes](https://www.w3.org/2023/Process-20230612/#revised-rec-substantive)

**Comment by AramZS** [comment]
There's a lot of activity around what used to be the Social WG's maintained recommendation, presumably that would indicate that there is interest in more active maintenance from a broad set of participants. The potential spec work listed is quite large and many of those could indeed use more active work.

**Comment by capjamesg** [comment]
I support the creation of a WG. There are several changes to be made to the ActivityPub specification that would require a WG. I thank @evanp for leading work as editor, and to the broader community -- particularly those involved with FEPs -- for helping to advance discussions on how ActivityPub can be improved.

There has also been interest in potentially working on jf2, Micropub, rel=me and IndieAuth. Any inclusion, however, would require an active editor (or, for larger specs, multiple editors) who have already worked on the specifications and can lend their expertise to the group.

I would prefer we have a single WG. I think we can work better as one group rather than managing cross-group communications, and we could have more social web experts in one place to support each other.

I have heard several concerns about new features in ActivityPub in particular but I don't know enough to have an opinion on whether AP should be allowed to have substantiative amendments in a prospective WG. With that said, a lot of work has been done on Micropub and IndieAuth that may introduce new functionality. If that is the case, I think denoting, by spec, whether the spec is in a "maintenance mode" or allows for larger changes would be beneficial. Thus, we could have ActivityPub in maintenance mode and IndieAuth, for example, in a more active editing mode.

I invite CG members ([participation is free](https://w3.org/community/socialcg/) to contribute to the [prospective WG charter](https://www.w3.org/wiki/SocialCG/WG_Charter_Discussion#Deliverables) wiki page, where we are collating thoughts for a charter. This page has been sent for distribution across the mailing list, [published on the w3.org website](https://www.w3.org/community/socialcg/2023/09/29/october-6th-meeting-and-calls-for-wg-scope-contributions/) and mentioned in meetings. I would love to see more people help us advance these discussions! (The wiki page is a forum for brainstorming and does not represent any final position of the group.)

**Comment by melvincarvalho** [comment]
There's been a few objections to a WG on the grounds of diversity and inclusion.

Looking at the wiki page, the following document is mentioned:  "Indieweb Living Standard"

https://indieauth.spec.indieweb.org/

Considering the significant shift of the fediverse towards the diverse open-source community, it seems prudent to adopt a living standard, supplemented by W3C snapshots that reflect the widely implemented practices.

**Comment by capjamesg** [comment]
The Living Standard process has worked well in the IndieWeb community. It was a great way for me to dip my toe in the water per se. I could participate without having to worry about what it meant to contribute to a standards body. My curiosity about implementing the standard was enough!

With that said, a Living Standard process is contingent upon there being active (existing) editors who are willing to triage feedback and weigh in on new ideas.

There is interest in propagating that the IndieWeb Living Standard work back up to the W3C. In the case of IndieAuth, there have been a few discussions on maturing the spec from the Living Standard to a full document intended for consideration as a W3C Recommendation per the Standards Track process.

**Comment by bumblefudge** [comment]
@melvincarvalho wrote:

> Considering the significant shift of the fediverse towards the diverse open-source community, it seems prudent to adopt a living standard, supplemented by W3C snapshots that reflect the widely implemented practices.

I _almost_ completely agree, in that I think preserving experimentation and extension "in the wild" should be a very high priority. I also recognize that understandably, hardening and refining details very carefully could facilitate the federation pipelines to scale up even more in the coming years and become core infrastructure of the web; Big Companies would like to see the refinements and iterations of AP and adjacent specs get to a point where they can Build Big and make it infrastructure. Those are two very hard priorities to balance. The balance of both priorities falls entirely on that mechanism whereby community consensus is achieved (or in the worst case, manufactured) for selecting which new features or use-cases steadily being adopted in the wild as candidates for inclusion or amalgamation onto the specification. It's easy to imagine it working well on a happy path, but it brings its own failure modes like any human endeavor: collusion, horse-trading, bias, in-group influence, technocratic groupthink, lack of development-philosophy diversity. 

More than any of those, I am more worried about scope creep, or the normative "laws" moving fast enough to break things many layers downstreams. As Evan pointed out today in a blog post, [some of those things are too important to risk breaking](https://evanp.me/2023/10/06/activitypub-the-socialcg-and-the-social-web/)

@capjamesg

> I would prefer we have a single WG. I think we can work better as one group rather than managing cross-group communications, and we could have more social web experts in one place to support each other. [...] I have heard several concerns about new features in ActivityPub in particular but I don't know enough to have an opinion on whether AP should be allowed to have substantiative amendments in a prospective WG.

The failure-mode I'm most worried about isn't that a joint working group would be too collaborative or too dispersed in its focus. It's that the precarious balance of today's adoption would be harmed by the temptation to start mandating things or combining things, paving the cow path not into a road but into a superhighway. The growth we're seeing in AP adoption is, in my mind, happening because of how thin a layer AP specifies and how this protects Activity data from taking the shape of a "one-product API".  I am adamant about protecting that "thin waist" of this nascent sociotechnology, which even 5 years ago could still be called a single "use case" and is now a whole swath of the web itself. 

One strategy for keeping AP simple is to specify it in isolation, zooming in on a subset of the use-cases of the SWICG (which, I never tire of repeating, would be great to publish before scoping anything else or even deciding on the operating agreements of today's SWICG). If cross-pollination is preferred to focus, that would be great and bring lots of value, but also its own risks and failure modes.  For instance, the small circle of people working halftime or more on these efforts could unwittingly bias the process towards this or that use-case, making this tradeoff or that tradeoff at the expense of flexibility and open-endedness. Collaborating closely could allow us to quickly advance *profiles* for how these disparate tools can work together, but without time to see how those adopt and affect market dynamics in the wild, bumping them too quickly form profiles to normative requirements fediverse-wide could have unforeseen consequences. 

On today's call, the idea was floated of putting a "gating function" (echoed above in Melvin's comment) into the scope of AP changes, to protect against the scope of AP-federation narrowing in ways that favor the short-term over the long. I think this is a good direction to explore.

Whether in one group or in more than one, whether in Working Groups or in Community Groups, the means matter more to me than the ends. I hope our iterations of all these specifications and technologies grows *more*, not *less* modular (or as the youngsters say these days, "composable") as we harden and iterate them, allowing more diverse architectures and form-factors to grow around the common language of Activity data.

**Comment by plehegar** [comment]
Update: the CG is still discussing this (somehow?). Until the CG reaches some conclusion, this won't move forward. Having said that, there doesn't seem to be urgency on this front.

**Comment by plehegar** [comment]
And, in case this wasn't clear before: in the absence of a WG, the [W3C ](https://www.w3.org/2023/Process-20231103/#team) will not republish the existing W3C Recommendations unless there is a decision from the CG to do so.

**Comment by plehegar** [comment]
The future group might want to consider adopting a [staged approach](https://github.com/w3c-fedid/Administration/blob/main/proposals-CG-WG.md) to move proposals from CG to WG .

**Comment by plehegar** [comment]
> Update: the CG is still discussing this (somehow?). Until the CG reaches some conclusion, this won't move forward. Having said that, there doesn't seem to be urgency on this front.

The conclusion the Team is waiting for from the CG is a simple expression that a Group charter is needed. We can work out the details of the charter after that.

**Comment by plehegar** [comment]
From [last year minutes](https://www.w3.org/2023/09/12-social-minutes.html#x726).

**Comment by plehegar** [comment]
Can we turn the [proposal](https://www.w3.org/wiki/SocialCG/WG_Charter_Discussion#Deliverables) into a charter?

**Comment by tantek** [comment]
[@github.com/plehegar](https://github.com/plehegar) I read that FedID CG/WG Proposal Stages proposal ([@w3c-fedid/Administration/blob/main/proposals-CG-WG.md](https://github.com/w3c-fedid/Administration/blob/main/proposals-CG-WG.md)) and it makes a lot of sense to me as a thoughtful and rational methodology to incubate ideas and proposals in a CG through levels and when to uplift a proposal into the corresponding WG. I support it.

I think this would work well for numerous technical proposals being discussed in the Social Web CG to help them evolve and advance iteratively, and provide a more explicit way to evaluate them (independent of their specific topic or technology) for taking up by a potential future Social Web WG.

(Originally published at: https://tantek.com/2024/269/t1/)

**Comment by tantek** [comment]
[@github.com/plehegar](https://github.com/plehegar) regarding:
&gt; Can we turn the proposal ([https://www.w3.org/wiki/SocialCG/WG_Charter_Discussion](https://www.w3.org/wiki/SocialCG/WG_Charter_Discussion)) into a charter?

I think that proposal made good progress on some sections of a potential future Social Web WG charter as of meetings and discussions 6-12 months ago.

However, given the CG/WG Proposal Stages you linked us to above, I think it would be worth splitting the large table of Deliverables into two tables for consideration:
* Specs published by the prior Social Web WG — for necessary maintenance
* New proposals or draft specs - for working through stages

Without objection, I can do the wiki-table editing to split up that draft table of deliverables so it matches better with the CG/WG Proposal Stages proposal.

(Please thumbs-up as encouragement if you support this)

(Originally published at: https://tantek.com/2024/269/t2/)

**Comment by plehegar** [comment]
ACTION: @plehegar to send advance notice to the AC.

**Comment by plehegar** [comment]
There are alternative initial proposals, to be discussed in CG: https://github.com/swicg/potential-charters/pull/2 and
https://github.com/swicg/potential-charters/pull/3

**Comment by plehegar** [comment]
CG is still iterating on its own CG charter, as well as the 2 drafts for a WG Charter. See discussion on [2024-11-08](https://github.com/swicg/meetings/tree/main/2024-11-08#minutes). Aim is to have a CG Charter in December and we could expect the WG Charter will come after that.

**Comment by plehegar** [comment]
still waiting for the CG to come up with a WG charter. If the CG is unable to propose a charter for the WG by the end of March, the W3C Team may come up with a proposed charter after that.

**Comment by plehegar** [comment]
CG is attempting to converge on the WG charter

**Comment by plehegar** [comment]
The CG converged on a charter:
  https://swicg.github.io/potential-charters/social-web-wg-charter.html

Copied at:
 https://w3c.github.io/charter-drafts/2025/social-wg.html

**Comment by plehegar** [comment]
Next steps:
1. Send advance notice to the AC
2. Update charter as needed with charter template
3. Request horizontal review of the charter
4. Find potential chairs

**Comment by plehegar** [comment]
[Advance notice](https://lists.w3.org/Archives/Member/w3c-ac-members/2025AprJun/0029.html)

**Comment by himorin** [comment]
Per we have a text `The mission of the Social Web Working Group is to maintain the W3C Recommendations` for mission and `This Working Group would maintain the Recommendations and Notes in the Social Web area.` in motivation and background, how about to change `The Working Group will deliver the following W3C normative specifications` (in 3.1) into will maintain like [past example of 2023 pmwg](https://www.w3.org/2023/06/pmwg-charter.html#normative)?

**Comment by matatk** [comment]
[No accessibility concerns from APA WG](https://www.w3.org/2025/06/18-apa-minutes.html#900f)

**Comment by simoneonofri** [comment]
No comments from SING, following-up @npdoty during updates it should be good to refresh the horizontals.

**Comment by himorin** [comment]
No comment or request from i18n

**Comment by gobengo** [comment]
tl;dr: There are many concerns about another SocialWG that have not been given due consideration, and the SocialCG has not decided whether there should be a SocialWG or what it's charter would be, in part due to legitimate concerns by the public and work item editors. There have not even been any concrete proposals in SocialCG, let alone CFCs, that can be objected to, and there certainly are no decisions from SocialCG according to it's relatively new (despite yearslong persistent efforts) decision policy. Please consider no SocialWG as an option (and one we are closest to continued convergence on) as well as, as an alternative to another SocialWG, the creation of a ActivityPub CG and/or [ActivityPub Maintenance WG](https://swicg.github.io/potential-charters/ap-wg-charter.html), which is the only suggestion we have from an editor of ActivityPub.

---

## SocialCG Has Not Decided on any Charter or even any WG

Point of information visavis @plehegar writing:

> [diff with charter proposed by CG](https://services.w3.org/htmldiff?doc1=https://swicg.github.io/potential-charters/social-web-wg-charter.html&doc2=https://w3c.github.io/charter-drafts/2025/social-wg.html)

> The CG converged on a charter:
> https://swicg.github.io/potential-charters/social-web-wg-charter.html

I have a concern that if I do not speak up, the record here is more than a bit misleading.

The CG has not decided in any way on a WG charter, nor has there been any particular CFC on a resolution that there should be another WG at all. In fact, there is a history of a lack of convergence on any particular charter or scope or whether there is consent to moving items from CG to a WG at all.

This link is to a repo the Chair created for 'potential charters'. Please find included there not only a potential Social Web WG Charter, but also an ActivityPub Maintenance WG Charter that is equally converged upon under SocialCG process.
https://swicg.github.io/potential-charters/ap-wg-charter.html

---

## Input from ActivityPub Editors

On the other hand, we have had [messages from an ActivityPub editor](https://lists.w3.org/Archives/Public/public-swicg/2023Oct/0001.html) supporting continued work in CGs and specifically an ActivityPub CG. I'd also like to highlight the specific concern that, one way or another, the combination of W3C WG Process, the Chairs, and oversight by W3C Staff, failed to maintain a healthy working environment leading the editor to remark that the W3C SocialWG "took a few years off my lifespan", a sentiment I share as well, and have explained in terms of the [cost in business value to the company I worked for during the first SocialWG](https://github.com/swicg/potential-charters/pull/3#issuecomment-2393594327).
> Speaking from experience in old SocialWG times, my company spent tens of thousands of dollars of time in SocialWG and related repos, and hearing about micropub and microfomats so much was completely irrelevant to our goal of delivering a w3c standardized ActivityStreams2 based solution to customers ASAP and had real business impact for us and our customers, and it added more than a year of delay to the SocialWG deliverable timeline. ActivityPub and Microformats stuff should have separate delivery timelines and not be able to hold each other up, and by far the most straightforward way of ensuring this is separate WGs and charters.

It does not seem be in the public interest for the W3C to ignore or overrule these concerns of an ActivityPub editor and past SocialWG participants without even an attempt to duly consider them, let alone without suggesting how any hypothetical WG might avoid the same mistakes and harms in the future, e.g. by ensuring different chairs and/or allowing for different groups for different approaches so as to avoid the quid-pro-quo dynamics we witnessed in the group from 2013-2018, which the minutes from then show affected group agendas, work items, and ultimately the design of security and privacy affordances cut during the transition from CR to TR, which affected the security and privacy considerations mentioned earlier by @npdoty.

---

## Timeline of Efforts to Recreate SocialWG + Concerns

Let me offer a different perspective:
* The chairs of SocialWG created SocialCG explicitly deciding that it would need no charter. Within minutes of the SocialCG being created back then, the public-swicg mailing list was turned off for all public discussion until [November 2022](https://lists.w3.org/Archives/Public/public-swicg/2022Nov/0000.html).
* At [TPAC 2023, there was an off-agenda day-of-f2f proposal](https://www.w3.org/2023/09/12-social-minutes.html#x726) to recharter SocialWG entertained by @plehegar and the former SocialWG Chairs, which likely would have been considered a resolution had I not almost missed a flight to dial in and raise the concern
    >  I noticed there was an agenda published before the meeting but it didn't include anything about rechartering
    > … wrt to process it would normally the agenda for f2f meetings would be published 2 weeks in advance, especially to discuss something as serious as rechartering a group
* Soon after I [proposed a decision policy](https://lists.w3.org/Archives/Public/public-swicg/2023Sep/0118.html) (based on w3c cg charter templates) for SocialCG as a CFC to the SocialCG Mailing List
  * It had objections from a former SocialWG chair, which I duly considered and addressed, leading to the withdrawal of any/all -1s
  * For some reason, a former unelected SocialCG chair (chosen by a former chair as a condition of their resigning) still assessed that there was a lack of conensus to adopt any decision policy based on consensus
  * There were no more CFCs that I'm aware of to adopt a decision policy for SocialCG
* At TPAC 2024, I spoke with @plehegar and a SocialCG chair, explaining that the former SocialWG Chairs did not believe there should be a charter or explicit decision policy for SocailCG, but I disagreed and so did the current SocialCG Chair, and that the SocailCG *cannot* decide on whether there should/shouldn't be a SocialWG, nor what kind of charter it would have, until SocialCG has a decision policy, e.g. part of the charter.
  * That seemed to make sense to plh who, I thought, agreed that SocialCG should have a decision policy for W3C Staff to look to when characterizing SocialCG decisions as awhole.
  * So the Chair and I worked to introduce a [potential charter for SocialCG](https://github.com/swicg/potential-charters/pull/1).
* On 2025-02-07, SocialCG Chair [sent a CFC to adopt the potential CG Charter](https://lists.w3.org/Archives/Public/public-swicg/2025Feb/0026.html) (which includes a decision policy that SocialCG makes decisions based on CFCs to the list), and (iiuc) there were no objections.
* There have been no CFCs on any WG Charters since then.
* I couldn't find any meeting minutes that reflect a charter discussion since then.

---

## On Pater-nizing Behavior

Lastly

> the CG is still discussing this (somehow?)

As I've mentioned, there have been numerous explicit and professional attempts in person at TPAC and on the SocialCG mailing list to raise concerns about the proposals to create a another SocialWG without any due concern for the harms and failures of the last one, including the failure to create any maintenance decision policy, leaving a [tyranny of structurelessness](https://en.wikipedia.org/wiki/The_Tyranny_of_Structurelessness) that we have only recently remedied by adopting a SocialCG charter by consensus.

This kind of language "(somehow?)", and the bigger pattern of misrepresenting SocialCG decisions and preferences of ActivityPub editors, elevating certain kinds of folks opinions over others, occurs to me as a bit 'patronizing' (to use the terminology of W3C [CEPC #21 or #14](https://www.w3.org/policies/code-of-conduct/)) of the work that so many have put in despite underdocumented structural challenges.

This is the exact type of unnecessary commentary that was unfortunately common in some SocialWG leadership before (sadly, meetings weren't recorded), and many of us have been working hard to ensure it does not continue to have undue influence over the future of ActivityPub and the broader social web.

Just so the wider public is clear on 'strategy':
* We're 7+ years into SocialCG without ever receiving a CFC advocating for a specific charter with specific rationale such that specific objections could even be raised. For anyone who does want another SocialWG, why not introduce a concrete proposal or CFC on the mailing list and facilitate public input and wide review?
* Is W3C staff (or the board?) willing to support either no SocialWG and continued work in SocialCG, or an ActivityPub Maintenance WG as described in the [potential charter](https://swicg.github.io/potential-charters/ap-wg-charter.html)? Work on ActivityPub in SocialCG is what SocialWG decided, and has in general been working quite well for wide participation.
* If not, will W3C Staff unblock the creation of an ActivityPub CG, which as I understand it has been blocked due to irresolution of whether there is consensus on a 'big tent' SocialWG? I'm speaking up here to refute the notion that there is consensus on a SocialWG, let alone any particular SocialWG Charter, as evidenced by the lack of any proposal on the matter visavis SocialCG's Chartered Decision Policy as well as the preemptive concerns raised by an ActivityPub editor.
* If not, can you explain more about how it is in the public interest to take work items like ActivityPub back behind a closed Social WG process without the consent of its Editors, and despite the business concerns I raised about how the initial SocialWG made it extremely expensive to contribute to AP/AS2 without also engaging or tolerating quid pro quos with the other work items in the group that have fundamentally different goals and design constraints?

**Comment by csarven** [comment]
The TAG recommends the following:

The charter should clarify what constitutes "maintenance" in the WG. For example:

* Whether the work will include [substantive changes](https://www.w3.org/policies/process/#substantive-change), e.g., based on errata, or any [registry changes](https://www.w3.org/policies/process/#registry-change), or [editorial changes](https://www.w3.org/policies/process/#editorial-change).
* Whether specifications are expected to update "Considerations" sections, e.g., Privacy, Security, Accessibility, Internationalization, Architectural, and IANA.
* Whether specifications will be revised in alignment with their Explainer content following the [Explainer Explainer](https://w3ctag.github.io/explainer-explainer/) guidance.

To ensure adequate coordination with other groups and organizations, the charter should include or clarify relevant coordination points. See issue: https://github.com/w3c/charter-drafts/issues/677 .

While this charter version focuses on maintaining prior work from the Social Web WG, it may be worth considering potential future items under Tentative Deliverables, aiming to bridge deliverables and improve interoperability.

No fundamental architectural concerns otherwise.

**Comment by plehegar** [comment]
fyi, https://trello.com/c/zMgcRehh/49-1700-1830-at-protocol

**Comment by plehegar** [comment]
There is a [CfC for the charter](https://lists.w3.org/Archives/Public/public-swicg/2025Sep/0001.html).

**Comment by plehegar** [comment]
Not seeing objections to the latest CfC, I requested that the charter be sent to the AC for review.

**Comment by dontcallmedom** [comment]
a few (hopefully mostly editorial) comments:
* the mission statement, motivation and scope all claim the group will do only maintenance, but since it has a tentative new deliverable (LOLA), that should probably be clarified - I assume the intent is that LOLA is the only CG deliverable that could be adopted without rechartering. The Recommendation-update-specific success criteria wouldn't work for that deliverable.
* the phrasing of the scope is a bit confusing: "SWICG acted as maintainer [...] This WG would maintain…". Maybe "This WG will bring consensus updates to the Recommendations", or "will take over maintenance of these Recommendations"
* I'm not sure a "substantive change" can (or should) "contain a section" on privacy and security; I think it's more likely that each substantive change with privacy and security implications would be reflected in a privacy & security considerations section
* the list of deliverables should add "Initial text" entries, with links to the dated versions of the Rec
* the CG/WG proposal stages still refer to Proposed Recommendations, which no longer exist
* "Workflow for Active Items" would probably be a better fit under "Participation" than under "Scope"
* that same subsection refers to "co-chair" elections, which I don't think are in line with the Process

**Comment by caribouW3** [comment]
In section 5.1, the paragraphs under Social Web CG is quite strange. The first sentence seems to suggest that implementors will gather in the CG. The second aragraph IIUC says that the CG will give approval to WG-proposed changes to the deliverables. (it is somewhat true, wrt Wide review, but not for individual decisions). I don't know what "The Chairs of this group should reject proposals that are incompatible with this Charter." really means. "Incompatible" there is an obscure terminology.

I propose:
- replace this part of the charter with something more boilerplate like "the CG is expected to review the deliverables produced by the WG."  and maybe "the Social Web CG will continue to incubate specifications that are not covered in this charter". 
- If we really need it, add a sentence about coordination with Implementors who do not participate in the WG.
- Use the Scope section to clarify "out of scope" proposals, otherwise any incubated proposal might be considered as in scope, therefore adoptable by the WG.

**Comment by dariusk** [comment]
@caribouW3 In typing up a previous draft of this response I realized that in an ActivityPub context at least, there are probably 4 big implementors we could get to join the WG and that might be sufficient, especially if we add in Invited Experts who are also implementors but are working on solo projects that are important but underfunded.

The intent is for implementors to join the WG for sure! When I said

> almost none of these people doing the work will be Invited Experts able to participate in a WG

I was referring to the plurality of implementors in both the ActivityPub and IndieWeb ecosystem who are essentially open source software enthusiasts working on personal projects. I was one of them for years, maintaining a software suite used by about 15,000 people worldwide as a solo dev. (I mention the number only to illustrate that a small solo project can still affect many people and be a notable-if-not-major player in a burgeoning ecosystem.) Only last year did I manage to get a job that lets me spend a percentage of my time on W3C and related standards work -- prior to that it was something I was interested in participating in but did not have the time and resources to do so. And most people working in the space covered by this charter are in that same boat.

So really I just meant to point out an economic reality, which is what we are trying to negotiate with the unusual-but-not-unprecedented setup of this charter. We'd like to give people a chance to participate and bubble up standards from the CG to the WG and to provide a two-way communication flow between them. I imagine something like:

* Proposal X from the CG is seeing a lot of adoption and there is discussion of making it a standard
* Proposal X gets discussed in the WG, perhaps the WG gives feedback to the CG
* After some back and forth, the WG brings on Person Y from the CG as an Invited Expert to be the Editor of a new specification draft for Proposal X
* From there, business as normal continues with an expectation of continued communication with the CG by the WG

**Comment by silverpill** [comment]
>>"almost none of these people doing the work will be Invited Experts able to participate in a WG."

>any reason for that? implementation and testing is using a lot of resources (I'd say much more than participating in the WG discussions).

@caribouW3 ActivityPub implementers often prefer [Fediverse Enhancement Proposals](https://codeberg.org/fediverse/fep/) process (FEP). It is more informal, and it's a "good enough" solution for our coordination problems.

Some implementers may be interested in participating in the WG, but if the current level of activity in the CG is a reliable indicator, there won't be many.

**Comment by dariusk** [comment]
> [The FEP process] is more informal, and it's a "good enough" solution for our coordination problems.

I would say it is a good enough solution for some coordination problems. FEPs are informal and excellent and also somewhat like an IETF Internet Draft, in that anyone can write and submit one. Unlike an Internet Draft, there is no "promotion" process to something more formal, and we sometimes end up with overlapping and conflicting FEPs. One use for something like this CG to WG pipeline is for things that require greater coordination than the publishing of a document. In fact that might be for me the main purpose of a WG: to provide specifications that an implementor can know were discussed thoroughly and fairly and reached consensus to the point where it's likely to be something stable to interop with.

**Comment by plehegar** [comment]
> Does the lack of expected completions on four deliverables mean that they are de-prioritized compared to the other three, or just that the completion cannot even be guessed at?

The former. The lack of expected completions means that the interest to update those is much lower but we wanted the charter to remain open to the possibility.

**Comment by plehegar** [comment]
[Disposition of comments](https://github.com/w3c/charter-drafts/issues?q=in%3Atitle%20%22wg%2Fsocial%22%20created%3A%3E2025-12-01) from AC Review.
